RM_SUI/RM_SUI/StoreTabBar/Tabs/ProductsView.swift

```diff
@@ -0,0 +1,145 @@
+        static let totalTextName = "Total:"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        VStack {
+            navigationBarView
+
+            Text("\(Constants.totalTextName) \(productsViewModel.totalSum)")
+                .font(.title)
+                .fontWeight(.bold)
+                .foregroundColor(.red)
+
+            ScrollView(showsIndicators: false) {
+                ForEach(productsViewModel.products, id: \.name) { product in
+                    furnitureItemView(product: product)
+                }
+            }
+            .padding(.horizontal, 30)
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var productsViewModel = ProductsViewModel()
+
+    private var backgroundGradientView: some View {
+        LinearGradient(
+            colors: Constants.backgroundGradient,
+            startPoint: .leading,
+            endPoint: .trailing
+        )
+    }
+
+    private var navigationBarView: some View {
+        ZStack(alignment: .bottom) {
+            backgroundGradientView
+            HStack {
+                ZStack {
+                    RoundedRectangle(cornerRadius: 25)
+                        .frame(height: 50)
+                        .padding()
+                        .padding(.horizontal, 40)
+                        .foregroundColor(.white)
+                    Label(Constants.searchLabelName, systemImage: Constants.searchLabelImageName)
+                        .font(.title)
+                        .foregroundColor(.indigo)
+                        .offset(CGSize(width: -40, height: 0))
+                }
+                NavigationLink {
+                    FilterView()
+                } label: {
+                    Image(systemName: Constants.filterButtonImageName)
+                }
+                .offset(CGSize(width: -30, height: 0))
+                .font(.largeTitle)
+            }
+        }
+        .ignoresSafeArea()
+        .frame(maxHeight: 80)
+    }
+
+    private func furnitureItemView(product: Product) -> some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 20)
+                .foregroundColor(Color(.systemGray5))
+            HStack {
+                Image(systemName: product.image)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: 50, height: 50)
+                    .foregroundColor(.red)
+
+                Spacer()
+                VStack {
+                    Text(product.name)
+                        .font(.body)
+                        .fontWeight(.bold)
+                    HStack {
+                        Button {
+                            productsViewModel.changeSellCountFor(
+                                value: 1,
+                                currentProduct: product
+                            )
+                        } label: {
+                            Text("+")
```

Comment:
```suggestion
                            Text()
```
Текст в константу и "-" тоже